server/routes/api/landing_page.py

```diff
@@ -39,6 +40,10 @@
 MIN_CHART_TO_KEEP_TOPICS = 30
 OVERVIEW = 'Overview'
 
+l10n = gettext.translation('chart_titles', localedir='l10n', languages=['es'])
```

Comment:
Could add an inline TODO here to replace es hardcoding with locale detection. Optional.

Reply:
done